src/widget/image-map/image-map.js

```diff
@@ -95,7 +95,7 @@ class ImageMap extends Widget {
 
                     // remove images in all languages
                     that.question.querySelectorAll( 'img' ).forEach( el => el.remove() );
-                    that.question.querySelector( '.option-wrapper' ).before( fragment );
+                    that.question.querySelector( 'div[class=option-wrapper]' ).before( fragment );
```

Comment:
Great find! 

This solution would be fragile, as it would break if we ever add another class to form option-wrapper. I'd prefer something like `('*:not(.widget) .option-wrapper')`.

Reply:
test the selector above but the result still the same
![new selector 1](https://user-images.githubusercontent.com/13865076/108029105-7e69d080-705f-11eb-8cec-520c2d1f876c.png)

since the image will be attached on the fieldset node, is it okay to use something like `('fieldset:not(.widget) > .option-wrapper')` or `('fieldset > .option-wrapper')` ?
![new selector 2](https://user-images.githubusercontent.com/13865076/108029306-cd176a80-705f-11eb-8f05-e0ea53d04b94.png)

Reply:
Thanks for checking! I think I understand why my suggestion doesn't work. It would be great to use the `widget` class to exclude it though...

Hmm... `[...document.querySelectorAll('.option-wrapper')].filter(el => !el.closest('.widget'))[0]` is maybe not very elegant.

Ok, let's just do `('fieldset > .option-wrapper')` then.

Reply:
no problem and code updated.